ooni/nettests/experimental/tls_handshake.py

```diff
@@ -743,6 +766,7 @@ def handshakeSucceeded(connection):
 
             self.report['host'] = host
             self.report['port'] = port
+            self.report['sni'] = self.sni
```

Comment:
I think the reason is that it uses the `getpeername()` method of `SSL.Connection` and this could be different than what is specified as an input, I think.

It's a while that I haven't looked at this code and I am not sure if this is actually desired behaviour. I suspect it's probably wisest to log both in different keys.